lib/ansible/cli/galaxy.py

```diff
@@ -199,59 +297,129 @@ def _display_role_info(role_info):
 
         return u'\n'.join(text)
 
+    @staticmethod
+    def _validate_collection_name(name):
+        if not is_collection_ref('ansible_collections.{0}'.format(name)):
```

Comment:
Fewer negations is easier to read.
```suggestion
        if is_collection_ref('ansible_collections.{0}'.format(name)):
        	return name

        raise AnsibleError("Invalid collection name, must be in the format <namespace>.<collection>")
```